kodilla-stream/src/test/java/com/kodilla/stream/portfolio/BoardTestSuite.java

```diff
@@ -135,4 +140,23 @@ public void testAddTaskListFindLongTasks() {
         //Then
         Assert.assertEquals(2, longTasks);
     }
+
+    @Test
+    public void testAddTaskListAverageWorkingOnTask() {
+        //Given
+        Board project = prepareTestData();
+
+        //When
+        List<TaskList> inProgressTasks = new ArrayList<>();
+        inProgressTasks.add(new TaskList("In progress"));
+        double averageWorkingOnTask = project.getTaskLists().stream()
+                .filter(inProgressTasks::contains)
```

Comment:
Po co Ci zbiór jednoelementowy? Nie możesz użyć zwykłego `"In progress".equals`?

Reply:
użyte :)
Tak bylo we wczesniejszych przykladach - w sumie pytanie czy w przypadku jakbym chcial powiekszyc zbior to wtedy musialbym przepisywac tez ta czesc, a inaczej tylko dodac wiecej do zbioru.

Reply:
Ogólnie tok myślenia dobry, ale trzeba podejść do tego bardziej "indywidualnie".
Jeśli projektujesz np. publiczne API, coś co będzie miało kontrakt z użytkownikami zewnętrznymi, i planujesz w przyszłości rozbudowywać w tym kierunku - rozsądnie jest od razu użyć kolekcji, niż pojedynczej wartości.
Tutaj jednak jest to całkowicie wewnętrzna sprawa (całość rozgrywa się wewnątrz metody, między linią 150 a 153) - jeśli będziesz chciał filtrować po dwóch wartościach, to i tak musisz grzebać w kodzie. Wtedy nie jest problemem zamiana wartości na kolekcję.